picatrix/lib/manager.py

```diff
@@ -32,7 +34,14 @@ class MagicManager:
   """Manager class for Picatrix magics."""
 
   MAGICS_DF_COLUMNS = ['name', 'description', 'line', 'cell', 'function']
+
   _magics: Dict[Text, Callable[[str, str], str]] = {}
+  _helpers: Dict[Text, Any] = {}
```

Comment:
You vary between Text and str in this file. Is there a reason for that? I'd assume we picked one and stick with it (although Text seems more pytype aligned).

Reply:
AFAICS
_helpers is a Dict[Text, Dict[Text, Somestuff]]

```
{
        'function': helper,
        'help': helper.__doc__.split('\n')[0],
        'types': typing_help,
    }
```

You can also turn this into explicitly typed record as a typing.NamedTuple or dataclass:

```
from dataclasses import dataclass

@dataclass
class Helper:
    function: Any
    help: Text
    types: Dict[Text, Any]
```

And then _helpers becomes Dict[Text, Helper].